docs/how-to-work-on-localized-client-webapp.md

```diff
@@ -93,7 +93,7 @@ You can test the client app in any language available in the [list of languages
 
 If you are testing a new language, create a folder with the language name as the title next to the other languages and copy the JSON files from another language into your new folder.
 
-Add the language to the `client` array as seen above in the [`config/i18n/all-langs.js`](https://github.com/freeCodeCamp/freeCodeCamp/blob/main/config/i18n/all-langs.js) file.
+Add the language to the `client` array as seen above in the [`config/i18n/all-langs.js`](https://github.com/freeCodeCamp/freeCodeCamp/blob/6b4a6a02568b809fc216ea8566ff5df446d1da4e/config/i18n/all-langs.js#L5) file.
```

Comment:
```suggestion
Add the language to the `client` array as seen above in the [`config/i18n/all-langs.ts`](https://github.com/freeCodeCamp/freeCodeCamp/blob/main/config/i18n/all-langs.ts) file.
```
That link appears to also be slightly outdated. The same case is for link on line `85`.

Reply:
I agree it's a terrible idea to use line numbers in docs because they will change all the time.

Reply:
I'd say so, as the file was migrated to ts, no reason to link to the older js version.